functions/venue.js

```diff
@@ -434,6 +434,13 @@ const removeAdmin = async (adminId) => {
     });
 };
 
+exports.createUser = functions.auth.user().onCreate(async (user) => {
```

Comment:
This is not being used anywhere in our code, you should add documentation about how this works and why it's here, otherwise someone will see no usage tomorrow and remove it.

Reply:
https://firebase.google.com/docs/functions/auth-events

@denisdimitrov This is a different type of function to most of the backend ones. There will be no usage in our code as it is an event trigger.

Reply:
Yes, I read the description of the PR. But it's not documented in the code itself. And when you see something that's not being used anywhere, you remove it....

Reply:
Added comment in 0774226691b386bb6d2e7e37fa6dc140838def6b